src/StructuredLogger/Progress.cs

```diff
@@ -9,6 +9,11 @@ public class Progress : IProgress<ProgressUpdate>
 
         public event Action<ProgressUpdate> Updated;
 
+        public virtual void Report(double ratio)

```

Comment:
Oops, I just removed it. Do you need it or could you use the ProgressUpdate?

Reply:
This method is not necessary but keeps the calling code more simple. Would you rather see it disappear?

Reply:
No, it’s fine, we can keep it